resources/js/exts.js

```diff
@@ -0,0 +1,5 @@
+/* 
+Extensions from coala-utils python library
+https://gitlab.com/coala/coala-utils/blob/master/coala_utils/Extensions.py 
```

Comment:
Line contains following spacing inconsistencies:
- Trailing whitespaces.

*SpaceConsistencyBear, severity NORMAL, section `spacing`.*

The issue can be fixed by applying the following patch:

```diff
--- a/resources/js/exts.js
+++ b/resources/js/exts.js
@@ -1,5 +1,5 @@
 /* 
 Extensions from coala-utils python library
-https://gitlab.com/coala/coala-utils/blob/master/coala_utils/Extensions.py 
+https://gitlab.com/coala/coala-utils/blob/master/coala_utils/Extensions.py
 */
 exts = ['XML', 'Markdown', 'C++', 'Ruby', 'TeX', 'Perl', 'TypeScript', 'Python', 'Cuda', 'PHP', 'CoffeeScript', 'Lua', 'JavaScript', 'C#', 'YAML', 'Objective-C', 'Fortran', 'OpenCL', 'HTML', 'VHDL', 'C', 'Go', 'R', 'SCSS', 'reStructuredText', 'Scala', 'SQL', 'CMake', 'Haskell', 'CSS', 'Matlab', 'Octave', 'Verilog', 'Java', 'Julia', 'JSON', 'Dart'];
```